std_msgs/Header       header
string                detid
float64               ts_sys
uint32		            counts
uint32[]              channel
uint32		            slow_avg
uint32		            fast_avg
uint32		            dose_rate
